sympy/core/tests/test_var.py

```diff
@@ -1,83 +1,91 @@
 # Tests for var are in their own file, because var pollutes global namespace.
-
+import textwrap
 from sympy import Symbol, var, Function, FunctionClass
 from sympy.utilities.pytest import raises
 
 # make z1 with call-depth = 1
 
 
-def _make_z1():
-    var("z1")
 
 # make z2 with call-depth = 2
 
 
-def __make_z2():
-    var("z2")
-
+def test_var():
+    exec(textwrap.dedent("""
```

Comment:
`textwrap.dedent` removes extra indentation. So, I guess, it's not needed here as you don't add extra indent to the code inside the `exec`